stable/polaris/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: 5.4.2
+version: 5.3.0
```

Comment:
I think this should be 5.5.0

Reply:
Oops, you're right. I semver-bumped backwards! *facepalm*